tests/test_wrappers.py

```diff
@@ -10,13 +10,14 @@
 import benchbuild.project as project
 import benchbuild.utils.compiler as compilers
 import benchbuild.utils.wrapping as wrappers
+from benchbuild.downloads.base import nosource
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'benchbuild.downloads.base'](https://app.codacy.com/manual/polyjit/benchbuild_2/pullRequest?prid=5694346)